src/components/Sponsors/SupportUsSection.js

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+
+import CheetoLeft from '../../images/cheetohs/cheetoLeft.svg';
+import CheetoRight from '../../images/cheetohs/cheetoRight.svg';
+import CustomButton from '../CustomButton';
+
+const useStyles = makeStyles((theme) => ({
+    divStyle: {
+        backgroundColor: '#4529AE',
+        alignItems: 'center',
+        textAlign: 'center',
+    },
+    title: {
+        color: '#FFFFFF',
+        paddingTop: '96px',
+        marginBottom: '16px',
+        fontFamily: 'Futura',
```

Comment:
You'll need to include this in gatsby-config in the fonts array for it to show up:
```
{
            family: `Futura`,
            variants: [`300`, `400`, `500`, `600`, `700`, `800`, `900`],
          }
```

Reply:
@demiwu96 I was talking to Rishi, maybe the font should be kept consistent regardless of screensize and what you saw on Figma may potentially be a mistake from design. Maybe u can use the same font for now.